files/phoneformat.js/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "phoneformat.js",
+  "repo": "albeebe/phoneformat.js",
+  "files": {
+    "basePath": "dist",
+    "include": ["phone-format.js", "phone-format.min.js", "phone-format.min.js.map"]
```

Comment:
Yes, but the `.map` file points to the unminified file (so will cause 404s)

https://github.com/albeebe/phoneformat.js/blob/master/dist/phone-format.min.js.map#L1